### PR TITLE
New appointment feedback fields

### PR DESCRIPTION
**Pull request by marcocolluramoj**
## What does this pull request do?

Updates the appointment feedback forms content and flow:

- Adds new sessionSummary, sessionResponse and sessionConcerns and attendanceFailureInformation fields.
- Replaces the confirmation page with a confirmation banner.

## What is the intent behind these changes?

To capture better session feedback.

### CODE REVIEW

File: server/routes/appointments/feedback/shared/viewFeedback/feedbackAnswersPresenter.test.ts [hunk 1]
```diff
@@ -121,60 +104,13 @@ describe(FeedbackAnswersPresenter, () => {
...
   describe('notifyProbationPractitionerAnswers', () => {
     describe('if the behaviour question was answered with yes', () => {
       it('returns an object with the question and answer given, converting the boolean value into a "yes"', () => {
         const appointment = actionPlanAppointmentFactory.build({
-          sessionFeedback: {
-            behaviour: {
-              behaviourDescription: 'Alex had a bad attitude',
+          appointmentFeedback: {
+            sessionFeedback: {
+              // behaviourDescription: 'Alex had a bad attitude',
```
Comment (rgforsyth) [inline server/routes/appointments/feedback/shared/viewFeedback/feedbackAnswersPresenter.test.ts:113]: Should we remove this?

**Comment by kth13** [comment]
I am ok with the overall change but only thing, AppointmentFeedback has sessionFeedback and attendanceFeedback. Why this needs to be AppointmentFeedback, can't this be just feedback?. I believe you are already inside the appointment object rt?

**Comment by kth13** [comment]
@marcocolluramoj I took a look again,  since its changing in lots of places and it is just the name change, please go ahead with appointmentFeedback, I will approve this

**Review by kth13 — APPROVED** [review]
LGTM

**Pull request merged**